src/main/scala-2.11-2.12/zio/prelude/InvariantVersionSpecific.scala

```diff
@@ -2,7 +2,7 @@ package zio.prelude
 
 import scala.collection.generic.CanBuildFrom
 
-trait TraversableVersionSpecific {
```

Comment:
Do we need to rename these? They deal exclusively with `Traversable` so that name seems to be more specific. We can still have the `Invariant` companion object extend it.

Reply:
I would feel uneasy about naming these `TraversableVersionSpecific`.
Using `*VersionSpecific` kinda like a hack, to support multiple incompatible Scala versions.

If we didn't support multiple versions, all the members would be in `Invariant`. But we want multiple Scala versions, so we need to put some members into `InvariantVersionSpecific`. But those members are still "logically" members of `Invariant`, so that's why I would prefer to reflect that in the `*VersionSpecific` trait name. Not doing that would likely confuse other people.

Reply:
Yeah, I was thinking more about the contents of the file, which while they are `Invariant` instances are more specifically `Traversable` instances. But I don't think it really matters.